forte/pymodule.py

```diff
@@ -196,7 +196,8 @@ def prepare_psi4_ref_wfn(options, **kwargs):
     # set DF and MINAO basis
     if 'DF' in options.get_str('INT_TYPE'):
         aux_basis = psi4.core.BasisSet.build(
-            molecule, 'DF_BASIS_MP2', options.get_str('DF_BASIS_MP2'), 'RIFIT', options.get_str('BASIS')
+            molecule, 'DF_BASIS_MP2', options.get_str('DF_BASIS_MP2'), 'RIFIT', options.get_str('BASIS'),
+            puream=wfn_new.basisset().has_puream()
```

Comment:
I am glad you caught this one.

Reply:
Alas, took me a day to sort things out. I am not sure this is the correct thing to do (i.e., should DF basis use cartesian even if JKFIT basis says spherical) but at least consistent.